### PR TITLE
fixes client/New to chat runtimes

### PR DESCRIPTION
**Pull request by silicons**

### CODE REVIEW

**Comment by SandPoot** [comment]
#15649

File: code/modules/client/client_procs.dm [hunk 1]
```diff
@@ -422,10 +422,12 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
...
 		GLOB.host = key
 		world.update_status()
-
+	
 	if(holder)
 		add_admin_verbs()
-		to_chat(src, get_message_output("memo"))
+		var/admin_memo_note = get_message_output("memo")
+		if(admin_memo_note)
+			to_chat(src, get_message_output("memo"))
```
Comment (deathride58) [inline code/modules/client/client_procs.dm]: why are we running get_message_output a second time here

Comment (deathride58) [inline code/modules/client/client_procs.dm]:
```suggestion
			to_chat(src, admin_memo_note)
```

**Review by deathride58 — CHANGES_REQUESTED** [review]

File: code/modules/client/client_procs.dm (hunk 1 above)
Comment (silicons) [reply code/modules/client/client_procs.dm]: oh true

**Comment by silicons** [comment]
@deathride58

**Pull request merged**